src/app/modules/filters/filters-content/filters-content.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, OnInit, EventEmitter, Output } from '@angular/core';
+
+@Component({
+  selector: 'ccf-filters-content',
+  templateUrl: './filters-content.component.html',
+  styleUrls: ['./filters-content.component.scss']
+})
+export class FiltersContentComponent implements OnInit {
+  filters: Record<string, unknown> = {};
+  @Output() applyFilters = new EventEmitter<Record<string, unknown>>();
```

Comment:
You should have an input and output here for filter just like selection (which maybe we call filter selection instead?), ie:
```
@Input() filter: Record<string, unknown> = {};
@Output() filterChange = new EventEmitter<Record<string, unknown>>();
```

Reply:
Actually, keep applyFilters to signal that apply was pressed, but still have filter/filterChange for binding outside this component.